idaes/generic_models/properties/tests/test_iapws95_with_harness.py

```diff
@@ -27,6 +27,8 @@ def configure(self):
         self.param_args = {"phase_presentation": iapws95.PhaseType.MIX}
         self.prop_args = {}
         self.has_density_terms = True
+        # Helmholt package initialization has not solver calls, so can't fail
```

Comment:
Typos: :Helmholt" -> "Helmholtz", "not" -> "no"

(here and 3 copy/paste locations below)